common/utilities/integration_adaptors_logger.py

```diff
@@ -36,7 +37,7 @@ def formatTime(self, record, datefmt):
     handler = logging.StreamHandler(sys.stdout)
 
     formatter = _CustomFormatter(
-        fmt='[%(asctime)sZ] %(message)s pid=%(process)d LogLevel=%(levelname)s ',
+        fmt='[%(asctime)sZ] %(message)s pid=%(process)d LogLevel=%(levelname)s LoggerName=%(name)s',
```

Comment:
Can I ask where this `LoggerName` came from? It seems to be a repeat information of the `ProcessKey` which holds the same value except it's concatenated with a process key number